packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts

```diff
@@ -159,12 +192,21 @@ export class SummarizerNode implements IRootSummarizerNode {
 			}
 		}
 
-		const result = await this.summarizeInternalFn(fullTree, true, telemetryContext);
-		this.wipLocalPaths = { localPath: EscapedPath.create(result.id) };
-		if (result.pathPartsForChildren !== undefined) {
-			this.wipLocalPaths.additionalPath = EscapedPath.createAndConcat(
-				result.pathPartsForChildren,
-			);
+		const result = await scheduleIdleTask(async (deadline: IdleDeadline) => {
```

Comment:
We should probably add this behind a feature flag to begin with since this has the potential to stall the summarization process and result in bugs. In the past, we have seen issues when summarization does not happen for a while (like 10k unsummarized ops).
We can also test this in the stress tests to see what the performance looks like.